test/integration/simple_instance/controls/gce.rb

```diff
@@ -0,0 +1,90 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project_id = attribute('project_id')
+zone = attribute('zone')
+instance_name = attribute('instance_name')
+network = attribute('network')
+subnetwork = attribute('subnetwork')
+vm_container_label = attribute('vm_container_label')
+
+control "gce" do
+  title "Google Compute Engine instance configuration"
+
+  describe command("gcloud --project=#{project_id} compute instances describe #{instance_name} --zone=#{zone} --format json") do
+    its('exit_status') { should be 0 }
+    its('stderr') { should eq '' }
+
+    let!(:metadata) do
```

Comment:
Where is this "metadata" name coming from. Suggest calling it something different (ex. `instance` or just `data`) since `metadata` has a specific meaning in this context.

Reply:
@morgante This is done in a few files -- I'd like to address this in a separate PR. I've added https://github.com/terraform-google-modules/terraform-google-container-vm/issues/8 to track it.